backtrader/stores/ccxtstore.py

```diff
@@ -105,8 +105,9 @@ def getposition(self, currency):
 
     @retry
     def create_order(self, symbol, order_type, side, amount, price, params):
-        return self.exchange.create_order(symbol=symbol, type=order_type, side=side,
+        ret_ord = self.exchange.create_order(symbol=symbol, type=order_type, side=side,
                                           amount=amount, price=price, params=params)
+        return self.fetch_order(ret_ord['id'])
 
```

Comment:
This one returned a CCXT parsed order. We use the `red_ord[id]` to fetch the parsed order. 

The reason for this is so that the returned order is has all of the CCXT unified attributes and is consistent across exchanges